jupyterhub/app.py

```diff
@@ -291,6 +292,15 @@ def _subdomain_host_changed(self, name, old, new):
             # if not specified, assume https: You have to be really explicit about HTTP!
             self.subdomain_host = 'https://' + new
 
+    domain = Unicode(
+        help="domain name without proto,port"
```

Comment:
Are we set on using 'proto'? My first reaction was "is this a typo for proxy?" Since it's only in 3 or so places in the code base, could we change it to 'protocol'?